src/System.Threading.Thread/src/System/Threading/Thread.cs

```diff
@@ -209,12 +305,14 @@ public void Resume()
         public static void BeginThreadAffinity() { }
         public static void EndThreadAffinity() { }
 
+#if !NETNATIVE
```

Comment:
LocalDataStore uses Dictionary<K,V>, which is not in System.Private.Corelib, so it won't compile.  I guess we need to add a reference to System.Collections, which we can do later.